### PR TITLE
Update (vehicle) abbreviations in IDs-Vehicle

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

Vehicle

**What?**

Resolve esoteric IDs in the fuel category

**Why?**

Provide better comparability across EFs

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -999,36 +999,36 @@ Energy,Fuel,fuel_type_lubricants,Lubricants,kg-CO2e/l,2.74972,,BEIS,2021,GB,2021
...
-Transport,Rail Travel,passenger_train-route_type_urban-fuel_source_average,National average for urban passenger train,kg-CO2e/pkm,0.03772874,,MfE,2020,NZ,2021/09/10,Emission intensity for urban passenger train (national average) including fuel consumption. Public transport passenger travel emission factors. Published by the Ministry for the Environment of New Zealand in 2020.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_consumption
+Transport,Rail Travel,passenger_train-route_type_london_underground-fuel_source_unknown,Underground passenger train,kg-CO2e/pkm,0.01434983,,MfE,2020,GB-LON,2021/09/10,Emission intensity for London underground passenger train including fuel consumption. Published by BEIS in June 2021.,https://environment.govt.nz/publications/measuring-emissions-detailed-guide-2020/,fuel_consumption
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv]: This entry is from the old version

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]
There are a couple of lines from the old version, otherwise looks great!!

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -1179,15 +1179,15 @@ Transport,Rail Travel,passenger_train-route_type_unknown-fuel_source_electricity
...
+Transport,Rail Travel,passenger_train-route_type_transit_rail-fuel_source_unknown,Transit rail passenger train,kg-CO2e/pmile,0.106,,EPA,2021,US,2021/08/23,Emission intensity for average passenger train. This emission factor accounts for transit rail: rail typically within an urban center such as subways/elevated railways/metropolitan railways (metro)/streetcars/trolley cars/tramways. Published by the EPA. The method of calculation used is the distance-based method which involves determining the mass/distance/mode of each shipment then applying the appropriate mass-distance emission factor for the vehicle used. LCA activity is unknown. This emission factor was updated in April 2021.,https://www.epa.gov/climateleadership/ghg-emission-factors-hub,unknown
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:1182]: Is this new or old? :) I am lost already

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -1042,17 +1042,17 @@ Transport,Vehicle,commercial_vehicle-vehicle_type_hgv-fuel_source_diesel-engine_
...
-Transport,Vehicle,commercial_vehicle-vehicle_type_lcv-fuel_source_electrolysis_hydrogen-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,Hydrogen LCV (electrolysis),kg-CO2e/km,0.311,50,ADEME,2020,EU,2021/08/20,Emission intensity of hydrogen (electrolysis) light commercial vehicle. Published by ADEME.,https://www.bilans-ges.ademe.fr/en/accueil,fuel_upstream_and_combustion
+Transport,Vehicle,commercial_vehicle-vehicle_type_lcv-fuel_source_electrolysis_fcev-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,Hydrogen LCV (electrolysis grid mix),kg-CO2e/km,0.091,50,ADEME,2020,FR,2021/08/20,Emission intensity of hydrogen (electrolysis using average electricity grid mix) light commercial vehicle. Published by ADEME.,https://www.bilans-ges.ademe.fr/en/accueil,fuel_upstream_and_combustion
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1045]: To align with other EFs this should fcev_electrolysis_ (fuel/power type first then provenence - e.g. this is hydrogen fuel cells (fcev) filled with hydrogen made from standard electrolysis)

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -1042,17 +1042,17 @@ Transport,Vehicle,commercial_vehicle-vehicle_type_hgv-fuel_source_diesel-engine_
...
+Transport,Vehicle,commercial_vehicle-vehicle_type_lcv-fuel_source_electrolysis_fcev_renewable-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,Hydrogen LCV (electrolysis renewable energy),kg-CO2e/km,0.082,50,ADEME,2020,FR,2021/08/20,Emission intensity of hydrogen (electrolysis using renewable energies mix) light commercial vehicle. Published by ADEME.,https://www.bilans-ges.ademe.fr/en/accueil,fuel_upstream_and_combustion
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1046]: fcev_electrolysis_renewable

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -1042,17 +1042,17 @@ Transport,Vehicle,commercial_vehicle-vehicle_type_hgv-fuel_source_diesel-engine_
...
+Transport,Vehicle,commercial_vehicle-vehicle_type_lcv-fuel_source_electrolysis_fcev-engine_size_unknown-vehicle_age_unknown-vehicle_weight_unknown,Hydrogen LCV (electrolysis),kg-CO2e/km,0.311,50,ADEME,2020,EU,2021/08/20,Emission intensity of hydrogen (electrolysis) light commercial vehicle. Published by ADEME.,https://www.bilans-ges.ademe.fr/en/accueil,fuel_upstream_and_combustion
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1047]: fcev_electrolysis_

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:1182]: The is old! This change should be removed

**Review by patstirling — CHANGES_REQUESTED** [review]
Super clean! Just a couple of issues with order of FCEV subIDs and a rogue rail travel backdate

**Review by patstirling — DISMISSED** [review]

**Review by patstirling — APPROVED** [review]

**Pull request merged**